apps/docs-v2/components/common/CommonHeader.tsx

```diff
@@ -0,0 +1,120 @@
+import {
+  Box,
+  Breadcrumbs,
+  Column,
+  Grid,
+  Typography,
+} from '@comfortdelgro/react-compass'
+import {usePageProps} from 'contexts/PageProps'
+import {useRouter} from 'next/router'
+import {useMemo} from 'react'
+
+export const CommonHeader = () => {
+  const {pageProps} = usePageProps()
+
+  const hasImage = Boolean(pageProps?.['imgSrc'])
+
+  if (
+    !pageProps?.backgroundColor &&
+    !pageProps?.backgroundImage &&
+    !pageProps?.imgSrc
+  ) {
+    return <></>
```

Comment:
I think should show default data

Reply:
No need, there is no default data to begin with